src/main/java/com/jfrog/ide/idea/scan/PypiScanner.java

```diff
@@ -86,7 +86,6 @@ private DepTree createSdkDependencyTree(Sdk pythonSdk) throws ExecutionException
                     continue;
                 }
                 node.getChildren().add(depId);
-                directDeps.remove(depId);
```

Comment:
I'd suggest changing the name of `directDeps` to something else. Maybe `dependencies`?
Also, the comment in line 69 needs to be updated with the new logic. It'd be best to explain why we treat all of the dependencies as direct.